__tests__/app.test.js

```diff
@@ -92,6 +92,28 @@ describe('\nGOOD ENDPOINTS\n', () => {
 			});
 		});
 	});
+	describe('/api/articles/:article_id/comments', () => {
+		describe('GET', () => {
+			test('Status 200: returns an array of comments for the given article ID', () => {
+				return request(app).get('/api/articles/1/comments').expect(200).then((response) => {
+					const comments = response.body.comments;
+					expect(comments).toEqual(expect.any(Array));
+				});
+			});
+			test('Status 200: returns an array with the correct properties', () => {
+				return request(app).get('/api/articles/1/comments').expect(200).then((response) => {
+					const comments = response.body.comments;
+					comments.forEach((comment) => {
+						expect(comment).toHaveProperty('comment_id');
+						expect(comment).toHaveProperty('votes');
+						expect(comment).toHaveProperty('created_at');
+						expect(comment).toHaveProperty('author');
+						expect(comment).toHaveProperty('body');
+					});
+				});
```

Comment:
need 200 but no comments related to the article. Can either return an empty array or return a message